platform_generator.R

```diff
@@ -107,11 +108,18 @@ if (TRUE %in% is.nan(SE_to_site_bandwidth$AvgBandwidth)) {
     max(transfers[transfers$UpDown==2,]$Bandwidth)
 }
 
+if (TRUE %in% is.na(SE_to_site_bandwidth$AvgBandwidth)) {
+  SE_to_site_bandwidth[is.na(SE_to_site_bandwidth$AvgBandwidth),]$AvgBandwidth <- 
+    max(transfers[transfers$UpDown==2,]$Bandwidth)
+}
+
```

Comment:
no need to replace is.nan by is.na as when there is no AvgBandwidth, the value is NaN (not a number).